packages/react-events/src/dom/testing-library/domEnvironment.js

```diff
@@ -20,10 +20,18 @@ export function hasPointerEvent() {
 }
 
 export function setPointerEvent(bool) {
-  const mock = bool ? emptyFunction : undefined;
-  global.PointerEvent = mock;
-  global.HTMLElement.prototype.setPointerCapture = mock;
-  global.HTMLElement.prototype.releasePointerCapture = mock;
+  const pointerCaptureFn = name => id => {
+    if (typeof id !== 'number') {
+      console.error(`A pointerId must be passed to "${name}"`);
```

Comment:
This makes sure the param type is correct because other the unit tests *not* would fail. And Flow also has a bug where the param type is incorrectly defined as `string`